cli/azd/pkg/azdext/azd_client.go

```diff
@@ -0,0 +1,91 @@
+package azdext
```

Comment:
```suggestion
// Copyright (c) Microsoft Corporation. All rights reserved.
// Licensed under the MIT License.

package azdext
```

Reply:
Same question from the other PR - what is the expectation here around copyright comments? We don't have a clear standard across the code base.